src/platform/plugins/shared/data/public/query/timefilter/types.ts

```diff
@@ -22,8 +22,8 @@ export interface TimefilterConfig {
 export type InputTimeRange =
   | TimeRange
   | {
-      from: Moment;
-      to: Moment;
+      from: Moment | string;
+      to: Moment | string;
```

Comment:
Is this necessary? It looks like `TimeRange` already defines these options to be strings.

```tsx
export type TimeRange = {
  from: string;
  to: string;
  mode?: 'absolute' | 'relative';
};
```

And setting the type explicitly with strings throws no type error.

<img width="427" alt="image" src="https://github.com/user-attachments/assets/270dceca-91e5-4dd8-b3c5-855fc2b8b361" />

Was this giving you a type error somewhere?

Reply:
Oh yeah I see now.

https://github.com/elastic/kibana/blob/342f620fa1f2dd1da51de606f3bf997acbe95bad/src/platform/packages/shared/kbn-es-query/src/filters/helpers/convert_range_filter.ts#L25-L28

This can be mixed (i.e. `{ from: string; to: Moment }`)

BTW you are QUICK!!!! 💨